cmd/node/node.go

```diff
@@ -447,7 +461,18 @@ func (app *SpacemeshApp) Start(cmd *cobra.Command, args []string) {
 	apiConf := &app.Config.API
 
 	validatorMock := sync.BlockValidatorMock{}
-	err = app.initServices(nodeID, swarm, "/tmp/", app.edSgn, bo, validatorMock, hareOracle, app.Config.LayerAvgSize)
+	poet, err := nipst.NewRemoteRPCPoetClient("127.0.0.1", 10)
+	if err != nil {
+		log.Error("poet server not found")
+	}
+
+	dbStorepath := "/tmp/"
```

Comment:
there is a config for that BaseConfig.DataDir
you can set the default instead of hardcoding